cpp/src/io/parquet/writer_impl.cu

```diff
@@ -1779,10 +1779,15 @@ auto convert_table_to_parquet_data(table_input_metadata& table_meta,
     std::fill_n(std::back_inserter(rg_to_part), num_rg_in_part[p], p);
   }
 
+  // Batch processing is no longer supported.
+  // This line disables batch processing (so batch size will no longer be limited at 1GB as before).
+  // TODO: All the relevant code will be removed in the follow-up work:
+  // https://github.com/rapidsai/cudf/issues/13440
```

Comment:
Not sure if this needs to be said, but please don't remove this code without further discussion. It could play a role in future performance improvements.